src/accessibilityIndex.jsx

```diff
@@ -2,20 +2,30 @@ import 'babel-polyfill'; // general ES2015 polyfill (e.g. promise)
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Provider } from 'react-redux';
+import { addLocaleData, IntlProvider } from 'react-intl';
+import enLocaleData from 'react-intl/locale-data/en';
 
 import AccessibilityPolicyPage from './components/AccessibilityPolicyPage';
 import store from './data/store';
 
+addLocaleData(enLocaleData);
+
+// hard-coded NON-defaults for testing. TODO: make these dynamic and correct
+const locale = 'en';
```

Comment:
Just checking - this is a "do this later" TODO and not a "do this as part of this PR" TODO, right?

Reply:
Correct, this will be part of the "get it working with platform" work in EDUCATOR-1613